README.md

```diff
@@ -204,6 +204,18 @@ Mandrill offers extra features on top of regular SMTP email like tagging, merge
 vars, and scheduling emails to send in the future. See
 [Bamboo.MandrillHelper](https://hexdocs.pm/bamboo/Bamboo.MandrillHelper.html).
 
+## Heroku Configuration
+
+If you are deploying to heroku, you will need to ensure that your configuration
+variables are available at compile time.
```

Comment:
It would be good to note that this is done with the `elixir_buildpack.config` file